src/mixins/editorMixin.js

```diff
@@ -185,6 +200,31 @@ export default {
 			}
 		},
 
+		scrollToLocationHash() {
+			if (document.location.hash) {
+				// scroll to the corresponding header if the page was loaded with a hash both in reader and viewer
+				const readerEl = document.querySelector('[data-collectives-el="reader"]')
+				const editorEl = document.querySelector('[data-collectives-el="editor"]')
+
+				let scrolled = false
+				for (const el of [readerEl, editorEl]) {
+					const headingEl = el?.querySelector(document.location.hash)
+					if (!headingEl) {
+						continue
+					}
+
+					headingEl.scrollIntoView({ behavior: 'instant' })
+					scrolled = true
+				}
+
+				if (scrolled === true) {
+					// Scroll up a bit as the heading is hidden behind the menubar otherwise
+					const scrollContainerEl = document.querySelector('.page-scroll-container')
+					scrollContainerEl.scrollBy({ top: -43, behavior: 'instant' })
+				}
```

Comment:
There are two alternatives to this:
* in text we scroll the element to the middle of the screen https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#block (block: 'center')
* There's the [scroll-margin css property](https://developer.mozilla.org/de/docs/Web/CSS/scroll-margin) when scrolling